plugins/signals-react/api-report.md

```diff
@@ -0,0 +1,25 @@
+## API Report File for "@backstage/plugin-signals-react"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+import { ApiRef } from '@backstage/core-plugin-api';
+import { JsonObject } from '@backstage/types';
+
+// @public (undocumented)
+export type SignalApi = {
+  subscribe(topic: string, onMessage: (message: JsonObject) => void): string;
+  unsubscribe(subscription: string): void;
```

Comment:
I think we could instead have the `subscribe` method return the unsubscribe method instead tbh, similar to observables, so `: { unsubscribe(): void }`. It's usually nice to be passing data around but in this case I think it's best to try to keep that bit internal.

Reply:
Good idea, makes this a bit easier to use.